src/c/main.c

```diff
@@ -720,90 +720,32 @@ static void ble_stop_handler (void *impl, bool force)
   free_conversions ();
 }
 
-static bool parse_args (int argc, char *argv[], int *args_position, const char *parameter_shortname,
-                        const char *parameter_longname, char **return_value)
+int main (int argc, char *argv[])
 {
-  if (strcmp (argv[*args_position], parameter_shortname) == 0 || strcmp (argv[*args_position], parameter_longname) == 0)
-  {
-    if (*args_position < argc - 1)
-    {
-      (*args_position)++;
-      *return_value = argv[*args_position];
-      (*args_position)++;
-      return true;
-    }
-    else
-    {
-      printf ("Option %s requires an argument\n", argv[*args_position]);
-      exit (0);
-    }
-  }
+  edgex_device_svcparams params = {"device-ble",
+                                   NULL,
+                                   getenv ("EDGEX_REGISTRY"),
```

Comment:
Should be NULL: The environment variable is `edgex_registry` (lowercase) and the SDK looks it up for us

Reply:
changed it to NULL